department-of-veterans-affairs/caseflow
# Refactor Related Tasks and Add New Related Task Form Component

Resolves [Create Logic to Have Appeal Tasks Appear in it's Own Section](https://jira.devops.va.gov/browse/APPEALS-34934)

# Description
- Simplify the Redux actions for related tasks and rework existing code to use simplified logic.

- Add skeleton for component for adding tasks to existing appeals.

## Acceptance Criteria

## Testing Plan
1. Go to the intake form.
2. Navigate to step 2 and toggle the "related to existing appeal" radio to "yes".
3. Select several appeals
4. A header for each selected appeal should appear/disappear below the table based on the related appeal's checked state.

# Frontend
## User Facing Changes

 BEFORE|AFTER
![before](https://github.com/department-of-veterans-affairs/caseflow/assets/2421172/0d995a4f-2822-4f95-820d-a19c5283a461)
|
![after](https://github.com/department-of-veterans-affairs/caseflow/assets/2421172/c5251c29-a20c-4ca1-bfaf-83fe3b816041)
